### ISSUE TITLE
Failed to activate the atom-material-ui theme

### ISSUE THREAD

**Issue by cssgirl** [atom-material/atom-material-ui#296]
1. In Themes menu, click gear icon to right of UI Theme dropdown after selecting "Atom Material Theme"
2. Scroll down to settings and click Primary color box. Select a new color (I selected a bright pink).

After that the error occurred and the UI styles were completely broken. I switched to another UI theme, and back to Atom Material Theme and my color changes were intact and the UI was looked as it should again.

**Atom Version**: 1.7.4
**System**: Mac OS X 10.11.5
**Thrown From**: [atom-material-ui](https://github.com/silvestreh/atom-material-ui) package, v1.2.13
### Stack Trace

Failed to activate the atom-material-ui theme

```
At missing opening `(` in /Users/ldinapoli/.atom/packages/atom-material-ui/styles/custom.less:2:3

LessError: missing opening `(`
  at /Users/ldinapoli/.atom/packages/atom-material-ui/styles/custom.less:2:3
```
### Commands

```
     -3:22.3.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-always.seti-ui-no-icons.use-animations.theme-solarized-dark-syntax.theme-atom-material-ui.has-custom-icons)
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "front-icons",
      "file-type-icons",
      "minimap-pigments"
    ],
    "themes": [
      "atom-material-ui",
      "solarized-dark-syntax"
    ]
  },
  "atom-material-ui": {
    "colors": {
      "abaseColor": {
        "red": 255,
        "green": 48,
        "blue": 108,
        "alpha": 1
      }
    }
  }
}
```
### Installed Packages

``` coffee
# User
atom-material-ui, v1.2.13
autoclose-html, v0.23.0
emmet, v2.4.3
file-icons, v1.7.13
pigments, v0.29.1
solid-completions, v2.0.0

# Dev
No dev packages
```

**Comment by silvestreh** [comment]
Did you set the color manually or by using the `<select>` with the color presets?

**Comment by silvestreh** [comment]
Just tried using both methods to change colors (presets and picking my own). With the packages you listed and without them. Can't reproduce this error.

You can reset the colors settings by opening Atom's dev console (<kbd>Cmd</kbd>+<kbd>Alt</kbd>+<kbd>i</kbd>) and entering the following snippet:

`atom.config.set('atom-material-ui.colors');`

Or, alternatively, you can open your Atom config file and get rid of the colors object:

```cson
# … 
"atom-material-ui":
    colors: # <-- this key right here
        abaseColor:
            red: 236
            green: 64
            blue: 122
            alpha: 1
        accentColor:
            red: 70
            green: 255
            blue: 193
            alpha: 1
        predefinedColor: "Pink"
# …
```

**Comment by cssgirl** [comment]
I had used the color wheel, clicking around until I found the color I liked best. I have tried this a few times as well with new colors and haven't been able to reproduce the error either.

**Comment by silvestreh** [comment]
Well, there was an issue with Atom's color picker where it would return invalid color values (both HEX and RGB) but I think it was resolved some time ago.

**Comment by silvestreh** [comment]
This _might be_ fixed in Material UI `2.0` (available for Atom 1.17-beta)
